heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/metricscache/MetricsCacheQueryUtils.java

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Twitter. All rights reserved.
```

Comment:
The current TMaster code uses protobuf objects internally during the query process. I would say protobuf api is not clunky. @kramasamy what's you opinion?